cl/scrapers/management/commands/cl_scrape_opinions.py

```diff
@@ -103,6 +103,7 @@ def make_objects(
         item.get("source") or Docket.SCRAPER,
         blocked=blocked,
         date_blocked=date_blocked,
+        appeal_from_str=item.get("lower_court_str", ""),
```

Comment:
why is this part of this PR

Reply:
Because we use that function to create the objects but currently does not allow saving lower court, that's why I had to update it